### DISCUSSION TITLE
Broken Top Languages Card

### DISCUSSION THREAD

**Discussion by panosru** [anuraghazra/github-readme-stats#1956] [Q&A] [answered]
**Describe the bug**
From what I see, only the top languages card is broken, while the URL works, the image card breaks when displayed through githubusercontent domain.

**Expected behaviour**
The card was working fine but now I see that it isn't, it should be displayed properly.

![image](https://user-images.githubusercontent.com/400362/185064492-3c1876cb-c304-44f3-a233-05dbbb9a9001.png)

**Screenshots / Live demo link (paste the github-readme-stats link as markdown image)**
![image](https://user-images.githubusercontent.com/400362/185064122-e3f0f777-6917-4417-bb9b-f0a85b32b5a7.png)

```
https://camo.githubusercontent.com/b24540ae7c0913401ebe8397adf515bbfcc5b9caf056304a4b16c8653fba3987/68747470733a2f2f6769746875622d73746174732d6769742d637573746f6d2d70616e6f7372752e76657263656c2e6170702f6170692f746f702d6c616e67732f3f757365726e616d653d70616e6f737275266c616e67735f636f756e743d3230266c61796f75743d636f6d7061637426636f756e745f707269766174653d7472756526686964655f626f726465723d74727565266c6f63616c653d656e266578636c7564655f7265706f3d6769746875622d726561646d652d73746174732c70616e6f7372752c20636f636b7069745f47524f5550532c6a616d657367656f7267653030372c686564797468656465762c6b61746572696e612d7765622c2e6e65742d726e642d6931386e2c7068702d63656e736f722c6672616d65776f726b2c4265747465725265666c656374696f6e2c646f636b65722d7068702d63656e736f722c70726f746f732c6e6f64652d6a696e6a732c70726f746f732d646f63732c4f78794e6f6465
```
produces: `Error Fetching Resource`

Code in use:
```html
<img align="right" src="https://github-stats-git-custom-panosru.vercel.app/api/top-langs/?username=panosru&langs_count=20&layout=compact&count_private=true&hide_border=true&locale=en&exclude_repo=github-readme-stats,panosru, cockpit_GROUPS,jamesgeorge007,hedythedev,katerina-web,.net-rnd-i18n,php-censor,framework,BetterReflection,docker-php-censor,protos,node-jinjs,protos-docs,OxyNode" alt="Top Langs" />
```

**Additional context**
I regenerated my developer token and redeployed vercel app just in case. I understand that this most likely is an issue with camo.githubusercontent.com, but, I wonder why it happens only with top languages card and if there is a workaround. Maybe related to #1002.

Thanks!

**Comment by rickstaa** [comment, marked as answer]
@panosru Thanks for your answer. The language card should work when you're using the latest main branch and deployed your Vercel instance following [this guide](git@github.com:anuraghazra/github-readme-stats.git). For the correct syntax, you can see my [README](https://github.com/rickstaa/rickstaa). 

I was able to fix your README using the following change https://github.com/panosru/panosru/pull/1 (see https://github.com/panosru/panosru/blob/acd92dd31729cfe8589467570bae377f72c012e8/README.md). Maybe the space messed things up. 🤔

**Comment by panosru** [comment]
@rickstaa, wow, so the two spaces were to blame! Thanks! I wasn't aware that they could cause the issue! Interesting to know!

**Reply by rickstaa** [reply]
No problem, I don't think the second space can cause this issue. The first space, however, is not allowed since URLS use [Percent-encoding](https://en.wikipedia.org/wiki/Percent-encoding), representing the space as the `%20` character.

**Reply by rickstaa** [reply]
I guess we could automatically change spaces to `%20` when more people encounter this problem.

**Reply by panosru** [reply]
yes, I must've accidentally placed the space after the comma and didn't even notice it, especially since it was on a new line on my screen width. Thank you for your time, very nice catch!
